Add mpsl fem power models (#8536)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

====================

* Fixed a build error that occurred when building an application for nRF53 SoCs with Simple GPIO Front-End Module support enabled.

> * Added the possibility to add custom models which split the requested TX power between the power on SoC output and the FEM gain in a way desired by the user.

Reviewer: ```suggestion
* Added the possibility to add custom models which split, as desired, the requested TX power between the power on SoC output and the FEM gain.
```
